users_app/tests/test_user.py

```diff
@@ -123,6 +124,13 @@ def test_login_success_with_Google_should_redirect_to_apsite(self):
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertEqual('supplier_app/ap-taxpayers.html', response.template_name[0])
 
+    def test_login_success_with_Google_should_redirect_to_buyersite(self):
```

Comment:
Good job! Don't know if it belongs to this PR but we should consider making test cases for the views that this new user is not allowed to get in.

Reply:
This new user uses urls that already have tests